client/my-sites/all-sites-icon/README.md

```diff
@@ -1,22 +0,0 @@
-All Sites Icon
```

Comment:
why'd this get dropped?

Reply:
Where would I put the README? `AllSitesIcon` is now an internal component of `AllSites`. It was never used independently. I'm not sure if it was ever supposed to.

There is still a README for the `block/all-sites` block. Do you think the deleted README contains some valuable info that should be copied to the parent component README? Or to a comment in code?

Reply:
ahhhhh i see. nevermind!